grouper/fe/forms.py

```diff
@@ -264,3 +265,12 @@ class UserTokenForm(Form):
     name = StringField("Token name", [
         validators.DataRequired(),
     ])
+
+
+class UserPasswordForm(Form):
+    name = StringField("Password name", [
```

Comment:
Separate passwords per application are definitely better, but we don't use this much, so I'm not sure how much effort it's worth putting into this.